CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Contract breaking changes:
 New functionality added in a backwards-compatible manner:
 
 * [OLMIS-2709](https://openlmis.atlassian.net/browse/OLMIS-2709): Changed ReferenceData facility service search endpoint to use smaller dto.
+* The /requisitions/requisitionsForConvert endpoint accepts several sortBy parameters. Data returned by the endpoint will be sorted by those parameters in order of occurrence. By defaults data will be sorted by emergency flag and program name.
```

Comment:
This new default order should be also documented in api-definition, probably also javadoc